js/eventloop.go

```diff
@@ -95,10 +95,10 @@ func (e *eventLoop) popAll() (queue []func() error, reserved bool) {
 }
 
 // start will run the event loop until it's empty and there are no reserved spots
-// or a queued function returns an error. The provided function will be the first thing executed.
+// or a queued function returns an error. The provided firstFunc will be the first thing executed.
 // After start returns the event loop can be reused as long as waitOnReserved is called
-func (e *eventLoop) start(f func() error) error {
-	e.queue = []func() error{f}
+func (e *eventLoop) start(firstFunc func() error) error {
```

Comment:
```suggestion
func (e *eventLoop) start(firstCallback func() error) error {
```
Now, after the naming thread, I'm thinking of having the same here could make it consistent

Reply:
done in [9a12345](https://github.com/grafana/k6/pull/2228/commits/9a12345e90be182fc271a33381f3c7a76e60d51f)